WcaOnRails/config/initializers/i18n_js.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# copied from https://github.com/fnando/i18n-js/blob/main/MIGRATING_FROM_V3_TO_V4.md
+Rails.application.config.after_initialize do
+  require "i18n-js/listen"
+  # This will only run in development.
+  I18nJS.listen
```

Comment:
Although the migration guide does mention to put that file here, I believe it would actually be executed in production (which the migration guide also strongly suggest not to do)!

Since we export the locales for production and tests, I'd suggest to either:
  - make sure this runs only in development (see the suggestion below)
  - move this part to `config/environments/development.rb`

```suggestion
  I18nJS.listen if Rails.env.development?
```

Since I'm not sure if I18nJS actually expects the configuration to be fully initialized, I believe my first suggestion should be the best/safest route!

Reply:
Yes, I agree. Although based on what the migration guide says, I'd expect them to run a similar "is Rails running in development mode?" check within the `listen` method that I'm calling here. But either way, your suggestion is the super safe route :)

Reply:
You're right actually, they [do check for that](https://github.com/fnando/i18n-js/blob/1e04f527eb683b7200db1fcd3cdbcdbba645575b/lib/i18n-js/listen.rb#L12)!
Weird to be doing that when the lib is not rails-specific but alright ^^